### DISCUSSION TITLE
希望用户某些场景下第一次打开的app不执行任何跳过逻辑

### DISCUSSION THREAD

**Discussion by bytemain** [gkd-kit/gkd#799] [Announcements]
### 新功能描述

遇到过好多次，新安装的某个软件要进行一些初始设置，但是却被规则命中直接跳过了初始设置。

希望可以设置项里加个选项

**Comment by lisonge** [comment]
你可以通过 通知栏快捷图标 临时关闭 无障碍功能

https://gkd.li/guide/external#shortcut

**Reply by bytemain** [reply]
coloros 15 点击磁贴没有用，app主页点击服务状态那个框也没有用，点一下只会弹个关闭无障碍的toast

**Reply by lisonge** [reply]
那是你的系统问题，使用测试版

https://github.com/gkd-kit/gkd/releases/tag/v1.9.3-beta.1

**Reply by lisonge** [reply]
https://gkd.li/guide/faq#unable_open_a11y

**Reply by bytemain** [reply]
试了下授予敏感权限，关闭权限监控以及新版本等，都不能在页面或者磁贴中关闭gld的功能，有什么日志我可以提供的吗 试了下logcat发现没有任何日志

**Reply by bytemain** [reply]
看了另外一个讨论，卸载后通过文件管理安装就可以关闭打开了.. 谢谢回复

**Reply by lisonge** [reply]
你说的这个另一个讨论的方式在我上面发给你的链接文档 <https://gkd.li/guide/faq#unable_open_a11y> 里面已经说明

**Reply by lisonge** [reply]
我是在我自己的小米设备上测试的，在访问受限制的情况下，可以主动关闭无障碍，但是不能主动开启无障碍

看来你的设备是在有限制情况下，主动开启或关闭无障碍都不行
